st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -165,6 +166,9 @@ def _get_output_values(self, exit_code, stdout, stderr, timed_out):
         else:
             error = None
 
+        if "InvalidStatusException" in stderr:
```

Comment:
One option would be to serialize it as a special result (like we do with regular result - we wrap it in a special start and end markers which are very unlikely to occur in regular output).